tests/unit/test_sparql_ds.py

```diff
@@ -4,3 +4,67 @@
 Author: Eugeniu Costetchi
 Email: costezki.eugen@gmail.com 
 """
+from urllib.error import URLError
+
+import pytest
+from SPARQLWrapper.SPARQLExceptions import QueryBadFormed, URITooLong
+
+from eds4jinja2.adapters.sparql_ds import SPARQLEndpointDataSource
+from tests import ENDPOINT_REMOTE_CORRECT, DUMMY_DESCRIBE_URI, SPO_LIMIT_10, WRONG_SPO_LIMIT_10, \
+    QUERY_LONGER_THAN_2048KB, ENDPOINT_LOCAL_CORRECT, ENDPOINT_INEXISTENT_SERVER, DUMMY_DESCRIBE_URI_GRAPH
+
+
+def test_connect_to_remote_endpoint():
+    fds = SPARQLEndpointDataSource(ENDPOINT_REMOTE_CORRECT)
+    response_text = str(fds.with_uri(DUMMY_DESCRIBE_URI)._fetch_tree())
+    assert len(response_text) > 2000
```

Comment:
looks like a too broad of an `assert`

Reply:
yeah, just wanted to know that the connection is performed successfully.